test/integrations/requirejs/test-requirejs.js

```diff
@@ -2,6 +2,17 @@ describe('Require.JS Pages', function() {
     it('loads mParticle properly', function() {
         var server = new MockHttpServer();
         server.start();
+        window.fetchMock.post('https://jssdks.mparticle.com/v3/JS/test_key/events', 200);
```

Comment:
Shouldn't this be using the `urls.events` variable?

Reply:
Yes, but I cant link to an expanded resolved comment, so I'll paste a screenshot here:
![image](https://user-images.githubusercontent.com/5377436/234957433-81be4b9b-a8ca-49d9-b389-74c3607d2a8d.png)

Reply:
Can you include a link to the Sus Eng ticket as a todo?

Reply:
done!